### PR TITLE
Update PRISM Functions Library.vbs

### PR DESCRIPTION
**Pull request by levesswe**
Adding two new functions for working with CORD documents... save_cord_doc and clear_line_of_text

### CODE REVIEW

**Comment by RobertFewins-Kalb** [comment]
Can you please alphabetize the functions?

**Comment by levesswe** [comment]
LOL!  I reckon I could…

Wendy LeVesseur
Child Support Officer

Anoka County Government Center
2100 3rd Avenue, Suite 401
Anoka, MN 55303
763-422-7323

**Comment by RobertFewins-Kalb** [comment]
Super dee duper. I know it's a little thing, but it makes navigating the funclib so much easier. Thanks, @levesswe!!

**Comment by levesswe** [comment]
Alrighty.  Should be good to go!

Wendy LeVesseur
Child Support Officer

Anoka County Government Center
2100 3rd Avenue, Suite 401
Anoka, MN 55303
763-422-7323

File: Shared Functions Library/PRISM Functions Library.vbs [hunk 1]
```diff
@@ -8,6 +8,13 @@
...
 'functionally complete at all times.
 '
+'
+'
+'****************************************************************************************
+'*******************KEEP LISTS IN ALPHABETICAL ORDER, PLEASE!!!***************************
+'****************************************************************************************
+'
+'
 'Here's the code to add, including stats gathering pieces (without comments of course):
```
Comment (RobertFewins-Kalb) [inline Shared Functions Library/PRISM Functions Library.vbs]: Love it!

**Comment by RobertFewins-Kalb** [comment]
Very nice! Thank you!!

**Pull request merged**